### DISCUSSION TITLE
Ebookjapan DRM

### DISCUSSION THREAD

**Discussion by ghost** [apprenticeharper/DeDRM_tools#1682] [Q&A]
Hi guys, just wondering if anyone can help me with this. Especially if anyone has experience with running scripts from github.

I have been searching around many different vendors for the highest quality manga (in Japanese), and found that ebook japan is the best available. However I am struggling to get this solution by another coder working.

https://github.com/y2361547758/EbixExtract

Ebookjapan use the .ebix format. Does anyone know how to run this? I have zero experience with anything this technical, and the walkthrough provided in the link doesn't make sense to me. Once I download the tools, where do I extract them to, how do I run it, etc etc.

Any help would be greatly appreciated

**Comment by ElleKayEm** [comment]
Why not ask over there?
